src/main/java/com/google/devtools/build/lib/rules/cpp/CcLinkingHelper.java

```diff
@@ -124,15 +126,15 @@ public CcLinkingOutputs getCcLinkingOutputs() {
   private final ActionRegistry actionRegistry;
   private final RuleErrorConsumer ruleErrorConsumer;
   private final SymbolGenerator<?> symbolGenerator;
+  @Nullable private final RuleContext ruleContext;
```

Comment:
why nullable?

Reply:
Because it can be null in case of sandwich, that we discussed offline yesterday. See this line: https://github.com/bazelbuild/bazel/pull/9267/files/49ca1c4f3364a34a398a03b39b29a83552ce1e69#r318975035